ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml

```diff
@@ -57,8 +57,8 @@ resources:
       cpu: '2'
       memory: 4Gi
     requests:
-      cpu: 100m
-      memory: 200Mi
+      cpu: 200m
```

Comment:
Just curious, what was the major reason behind this?  OOMkills? Slow? gut feeling? For fun?

Reply:
SLOW... super slow when executing multiple `go runs`